demo/start.sh

```diff
@@ -56,26 +63,28 @@ for port in 8080 8081 8082; do
 		        compress: false
 		PORTLISTENERS
 		)
-        echo "${listeners}" >> $DIR/etc/$port.config
-
-        # Disable tls for the servers
-        printf '\n\n# Disable tls on the servers.' >> $DIR/etc/$port.config
-        echo '# DO NOT USE IN PRODUCTION' >> $DIR/etc/$port.config
-        echo 'use_insecure_ssl_client_just_for_testing_do_not_use: true' >> $DIR/etc/$port.config
-        echo 'federation_verify_certificates: false' >> $DIR/etc/$port.config
-
-        # Set tls paths
-        echo "tls_certificate_path: \"$DIR/etc/localhost:$https_port.tls.crt\"" >> $DIR/etc/$port.config
-        echo "tls_private_key_path: \"$DIR/etc/localhost:$https_port.tls.key\"" >> $DIR/etc/$port.config
+        {
+            echo "${listeners}"
```

Comment:
why not replace this with a simple `cat` ?

Reply:
the reason to use the group command instead of cat is that the 
cat is a binary written in c even with a great performance the performance and readability of the group command is better, in this situation the bash group all commands and redirect the result to the file, and the cat is a binary written for the manipulation of files.

## References
[https://mywiki.wooledge.org/BashFAQ/014](https://mywiki.wooledge.org/BashFAQ/014)
[https://www.man7.org/linux/man-pages/man1/cat.1.html](https://www.man7.org/linux/man-pages/man1/cat.1.html)

Reply:
I think you misunderstand me.

Currently, there is a `cat` command (`cat <<-PORTLISTENERS`) whose output is written to a bash variable, which is then `echo`ed again. I'm asking why not skip capturing the output, and run the `cat` from inside the command group so that its output goes straight to  the output file.

Reply:
sorry for my mistake I was thinking of using the cat to redirect the output to the file